release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -341,6 +341,49 @@ The Cloud Credential Operator utility (`ccoctl`) now creates secrets that use re
 
 With this release, when you xref:../installing/installing_gcp/uninstalling-cluster-gcp.adoc#cco-ccoctl-deleting-sts-resources_uninstalling-cluster-gcp[delete GCP resources with the Cloud Credential Operator utility], you must specify the directory containing the files for the component `CredentialsRequest` objects.
 
+[discrete]
+[id="ocp-4-12-psa-restricted-enforcement"]
+=== Future restricted enforcement for pod security admission
+
+Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.
+
+Global restricted enforcement for pod security admission is currently planned for the next minor release of {product-title}. When this restricted enforcement is enabled, pods with pod security violations will be rejected.
+
+See xref:../authentication/understanding-and-managing-pod-security-admission.adoc#security-context-constraints-psa-alert-eval_understanding-and-managing-pod-security-admission[Identifying pod security violations] for information about how to find which workloads are causing pod security violations.
+
+// TODO: What "call to action" can we say after this for people to prep for 4.13 enforcement? I don't see any docs on what to actually do when you have a workloads in violation
+
+// Resources
+// - https://connect.redhat.com/blog/important-openshift-changes-pod-security-standards
+// - https://docs.google.com/document/d/1fL8kDkAGejFM335AwjFL2A0keIKbQEQ2kZYNVL7FTiU/edit#
+
+To prepare for this upcoming change, ensure that your workloads can run as restricted. Workloads that are not configured according to the enforced security standards defined globally or at the namespace level will be rejected. The `restricted-v2` SCC admits workloads according to the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Restricted] Kubernetes definition.
```

Comment:
@stlaz Okay so here is my attempt at the initial "call to action" I was looking for. Let me know what you think of this.